backoffice/modules/catalog/models/Product.ts

```diff
@@ -10,6 +10,7 @@ export type Product = {
   gtin: string;
   slug: string;
   price: number;
+  metaTitle?: string;
```

Comment:
I am trying to understand why this field is optional.

Reply:
i resolved all your comment